Recipes/CGC/files_copyFromMyProject.ipynb

```diff
@@ -107,7 +107,7 @@
     "\n",
     "# LIST all files in the source and target project\n",
     "my_files = api.files.query(limit = 100, project = my_project.id).all()\n",
-    "source_files = api.files.query(limit = 100, project = source_project.id).all()\n",
+    "source_files = list(api.files.query(limit = 100, project = source_project.id).all())\n",
```

Comment:
So @SenadI specifically advised me not to use list() unless it was really needed.

The list-comprehension for **source_file** used to work well with the generator object, is it broken now or some other use I've missed?